src/02_spec/07_modules/dem_uart/index.rst

```diff
@@ -0,0 +1,79 @@
+UART Device Emulation Module (DEM-UART)
+=======================================
+
+.. figure:: img/osd_dem_uart.*
+   :alt: Device Emulation Module UART
+
+The UART Device Emulation Module connects to the bus of a given CPU on one side, and to the Debug Interconnect on the other.
+Towards the CPU it behaves like a UART-16550A device, but instead of transmitting information over a UART-Interface it instead passes it to a host PC via the Debug Interconnect.
+
+.. _uart-registers:
+
+16550 UART Registers
+--------------------
+The following UART registers are implemented and accessible via the bus, the address mapping is in accordance with the UART-16550(A) standard as specified in this `Datasheet <http://caro.su/msx/ocm_de1/16550.pdf>`_.
+No FIFOs are present in hardware.
+No Modem or DMA-Mode related features, registers or interrupts are implemented.
+Writing to a register that is not implemented has no effect, reading from such a register will always return ``0x00``.
+
+All registers are 8 bit wide.
+
+.. tabularcolumns:: |p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.40\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|
```

Comment:
You need to have the latex columns in sync with the widths below, otherwise the latex build fails.

```patch
-.. tabularcolumns:: |p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.40\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|
+.. tabularcolumns:: |p{\dimexpr 0.10\linewidth-2\tabcolsep}|p{\dimexpr 0.10\linewidth-2\tabcolsep}|p{\dimexpr 0.20\linewidth-2\tabcolsep}|p{\dimexpr 0.10\linewidth-2\tabcolsep}|p{\dimexpr 0.50\linewidth-2\tabcolsep}|
```